README.md

```diff
@@ -362,8 +363,8 @@ if (shaderResult1.result != VK_SUCCESS)
 }
 
 // std::tie support.
-vk::Result result;
-vk::ShaderModule shaderModule2;
+Result result;

```

Comment:
I think, instead of removing `vk::` here, we should add it in the above snippets.